test/spec/provider/camunda/element-templates/cmd/ChangeElementTemplateHandlerSpec.js

```diff
@@ -2664,93 +2887,539 @@ describe('element-templates - ChangeElementTemplateHandler', function() {
+          scopes: [
+            {
+              id: 'error-1',
+              type: 'bpmn:Error',
+              properties: [
                 {
-                  $type: 'camunda:InputParameter',
-                  name: 'input-1-name',
-                  value: 'input-1-changed-value'
-                }
-              ],
+                  value: 'error-code',
+                  binding: {
+                    type: 'property',
+                    name: 'errorCode'
+                  }
+                },
+                {
+                  value: 'error-message',
+                  binding: {
+                    type: 'property',
+                    name: 'camunda:errorMessage'
+                  }
+                },
+                {
+                  value: 'error-name',
+                  binding: {
+                    type: 'property',
+                    name: 'name'
+                  }
+                }
+              ]
+            }
+          ]
+        };
+
+        var newTemplate = {
+          properties: [
+            {
+              value: 'error-expression-new-value',
+              binding: {
+                type: 'camunda:errorEventDefinition',
+                errorRef: 'error-1'
+              }
+            },
+          ],
+          scopes: [
+            {
+              id: 'error-1',
+              type: 'bpmn:Error',
+              properties: [
+                {
+                  value: 'error-code',
+                  binding: {
+                    type: 'property',
+                    name: 'errorCode'
+                  }
+                },
+                {
+                  value: 'error-message',
+                  binding: {
+                    type: 'property',
+                    name: 'camunda:errorMessage'
+                  }
+                },
+                {
+                  value: 'error-name',
+                  binding: {
+                    type: 'property',
+                    name: 'name'
+                  }
+                }
+              ]
+            }
+          ]
+        };
+
+        changeTemplate(serviceTask, oldTemplate);
+
+        var errorEventDefinition =
+          findErrorEventDefinition(serviceTask, 'error-1');
+
+        updateBusinessObject(serviceTask, errorEventDefinition, {
+          expression: 'error-expression-updated-value'
+        });
+
+        // when
+        changeTemplate(serviceTask, newTemplate, oldTemplate);
+
+        // then
+        var errors = findRootElementsByType(getBusinessObject(serviceTask), 'bpmn:Error'),
+            error = errors[0];
+
+        expect(errors).to.have.length(1);
+
+        expect(error).to.exist;
+        expect(error.get('id').indexOf('error-1')).to.equal(0); // start with binding error ref
+        expect(error.get('errorCode')).to.equal('error-code');
+        expect(error.get('name')).to.equal('error-name');
+        expect(error.get('errorMessage')).to.eql('error-message');
```

Comment:
Should we not have an additional expect here?
`expect(errorEventDefinition.get('expression')).to.eql('error-expression-updated-value');`